src/lib/configuration/config.py

```diff
@@ -42,18 +42,32 @@ def release_tag():
 
 
 def gcp_metadata_url():
+    DEV_URL = os.environ.get("DEV_URL", None)
+    if DEV_URL:
+        return "http://localhost:8080/metadata.google.internal/computeMetadata/v1"
+
```

Comment:
Please avoid making changes to dynatrace-gcp-monitor code. Same thing can be achieved by `export GCP_METADATA_URL = 'http://localhost:8080/metadata.google.internal/computeMetadata/v1'`